lib/modules/manager/terraform/lockfile/hash.ts

```diff
@@ -107,8 +107,8 @@ export class TerraformProviderHash {
   }
 
   @cache({
-    namespace: `datasource-terraform-provider-hash`,
```

Comment:
The idea is that in future we'll infer cache namespace from the class name, by transforming it from `TerraformProviderHash` to `terraform-provider-hash`